mappings/net/minecraft/client/gui/MultilineTextField.mapping

```diff
@@ -0,0 +1,164 @@
+		ARG 0 text
+	METHOD m_opbgamoe getLines ()Ljava/lang/Iterable;
+		COMMENT {@return the lines of this text field's text}
+	METHOD m_qgrxeglv setSelecting (Z)V
+		COMMENT Sets whether this text field is currently selecting.
+		COMMENT <p>
+		COMMENT If using the widget, this is done by dragging or holding down Shift and clicking.
+		ARG 1 selecting
+			COMMENT {@code true} if this text field is currently selecting, or {@code false} otherwise
+	METHOD m_qlrljnzk getCursor ()I
+	METHOD m_satfxbxr getPreviousWordAtCursor ()Lnet/minecraft/unmapped/C_krhsyzys$C_ugdorfqq;
+		COMMENT {@return the substring of a word whose start position is before the cursor}
+		COMMENT <p>
+		COMMENT A word is a string consisting entirely of non-whitespace characters.
+		COMMENT If the cursor is in the middle of a word, the start position is that of the word;
+		COMMENT if not, the start position is that of the first word before the cursor.
+		COMMENT
+		COMMENT @see #getNextWordAtCursor()
+	METHOD m_sbsgitud getLine (I)Lnet/minecraft/unmapped/C_krhsyzys$C_ugdorfqq;
+		COMMENT {@return the line with index {@code index}}
+		ARG 1 index
+			COMMENT the line index
+	METHOD m_sebfshwr (Lnet/minecraft/unmapped/C_cpwnhism;II)V
+		ARG 1 text
+		ARG 2 start
+		ARG 3 end
+	METHOD m_snbipear getCurrentLineIndex ()I
+	METHOD m_tzncxvou moveCursor (Lnet/minecraft/unmapped/C_rvckgptc;I)V
+		COMMENT Moves the cursor by {@code amount} characters.
+		COMMENT
+		COMMENT @apiNote See {@link CursorMovement} for the types of the movement.
+		ARG 1 movement
+			COMMENT the movement of the cursor
+		ARG 2 amount
+	METHOD m_uecuspsi setCursorMoveListener (Ljava/lang/Runnable;)V
+		COMMENT Sets the cursor move listener that is called every time the cursor position changes.
+		ARG 1 listener
+			COMMENT the listener that runs when the cursor position changes
+	METHOD m_uhklhrmz hasSelection ()Z
+	METHOD m_uwuewaqs rewrap ()V
+	METHOD m_vuhkjybg getMaxLength ()I
+		COMMENT {@return the maximum length of this text field in characters}
+		COMMENT <p>
+		COMMENT If this equals {@link #UNLIMITED_LENGTH}, the text field does not have a length limit.
+		COMMENT Edit box widgets with such text fields do not show the current text length indicator.
+		COMMENT
+		COMMENT @see #setMaxLength(int)
+		COMMENT @see #hasMaxLength()
+	METHOD m_wbcaxtdn truncate (Ljava/lang/String;)Ljava/lang/String;
+		ARG 1 text
+	METHOD m_wtqoabei getWordEndIndex (I)I
+		COMMENT {@return the end index of the word starting at {@code startIndex}}
+		COMMENT <p>
+		COMMENT A word is a string consisting entirely of non-whitespace characters.
+		COMMENT Therefore, the end index is the index of the character whose succeeding character
+		COMMENT is the first whitespace since {@code startIndex}.
+		ARG 1 startIndex
+			COMMENT the index where the word starts
+	METHOD m_xwekzcbx onChange ()V
+	METHOD m_zabiisny hasMaxLength ()Z
+		COMMENT {@return {@code true} if this text field has a maximum length limit, or {@code false} otherwise}
```

Comment:
`maximum length limit` is a bit redundant imo
```suggestion
		COMMENT {@return {@code true} if this text field has a maximum length, or {@code false} otherwise}
```